bevy_lint/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+//! <style>
+#![doc = include_str!("../assets/rustdoc.css")]
+//! </style>
+//!
```

Comment:
you should use https://doc.rust-lang.org/rustdoc/command-line-arguments.html#-e--extend-css-extend-rustdocs-css